.nuspec

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2013/05/nuspec.xsd">
+    <metadata>
+        <id>Messerli.QueryProvider</id>
+        <version>0.1.0</version>
+        <authors>Jan Nils Ferner, Dominic Groth, Thomas Bruderer</authors>
+        <owners>messerli-informatik-ag</owners>
+        <projectUrl>http://github.com/messerli-informatik-ag/query-provider</projectUrl>
+        <license type="expression">MIT or Apache-2.0</license>
+        <description>A generic LINQ QueryProvider implementation, used by LinqToRest.</description>
+        <copyright>Copyright ©2019 Messerli Informatik Ag</copyright>

```

Comment:
AG

Reply:
Addressed in https://github.com/messerli-informatik-ag/query-provider/pull/1/commits/9b871ce91afc0cfe943f2661a17ed5bbc8b2a530